### PR TITLE
Adding long descriptions to commands and flags - draft messages that'l…

### PR DESCRIPTION
**Pull request by rmohan20**
…l be fixed soon

### CODE REVIEW

File: messages/add.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+  "commandDescriptionLong": `Add custom rules to Sfdx Scanner's registry to run them along with built-in rules. 
+  Rules should have been compiled and tested separately. 
+  Please refer to our Custom Rules help page for more information: 
+  <link to Custom Rules Help Page>`,
   "flags": {
-    "languageDescription": "Language against which the custom rules will evaluate.",
-    "pathDescription": `One or more paths to custom rule definitions. Specify multiple values with a comma-separated list.`
+    "languageDescription": "language against which the custom rules will evaluate",
+    "languageDescriptionLong": "Language against which the custom rules will evaluate.",
+    "pathDescription": "file path(s) to custom rule JAR files",
```
Comment (michaelhoefer) [inline messages/add.js:10]: These are not necessarily Jar files though, are they?  We should be supporting just directories with .class files in them too, which is why the description before did not mention just Jars.

File: messages/describe.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 module.exports = {
   "commandDescription": "Provide detailed information about a rule.",
+  "commandDescriptionLong": `Use this command to understand an individual rule better.
+  For each rule, you can find information about the language it works on, 
+  the violation it detects as well as an example code of how the violation looks.
+  The description also includes the categories and rulesets that the rule belongs to.
+  Please make sure your machine has Java8 or greater setup correctly.`,
```
Comment (michaelhoefer) [inline messages/describe.js:7]:
```suggestion
  Please make sure your machine has Java 8 or greater setup correctly.`,
```

File: messages/describe.js [hunk 3]
```diff
@@ -1,7 +1,13 @@
 module.exports = {
   "commandDescription": "Provide detailed information about a rule.",
+  "commandDescriptionLong": `Use this command to understand an individual rule better.
```
Comment (michaelhoefer) [inline messages/describe.js:3]:
```suggestion
  "commandDescriptionLong": `Use this command to better understand a particular rule.
```

File: messages/describe.js [hunk 4]
```diff
@@ -1,7 +1,13 @@
...
+  For each rule, you can find information about the language it works on, 
+  the violation it detects as well as an example code of how the violation looks.
+  The description also includes the categories and rulesets that the rule belongs to.
+  Please make sure your machine has Java8 or greater setup correctly.`,
   "flags": {
-    "rulenameDescription": "The name of a rule."
+    "rulenameDescription": "The name of a rule.",
+    "rulenameDescriptionLong": "Name of the rule whose description you would like to view."
```
Comment (michaelhoefer) [inline messages/describe.js:10]:
```suggestion
    "rulenameDescriptionLong": "Name of the rule to describe in more detail."
```

File: messages/list.js [hunk 5]
```diff
@@ -1,9 +1,16 @@
 module.exports = {
   "commandDescription": "Lists basic information about all rules matching provided criteria",
+  "commandDescriptionLong": `Lists all the rules available in the catalog. To look at a smaller set of rules at a time,
```
Comment (michaelhoefer) [inline messages/list.js:3]:
```suggestion
  "commandDescriptionLong": `Lists all the rules available in the catalog. To look at a smaller set of rules,
```

File: messages/list.js [hunk 6]
```diff
@@ -1,9 +1,16 @@
...
+  use the filter options available. To get more information about a specific rule,
+  you can use the scanner:rule:describe command.
+  Please make sure your machine has Java8 or greater setup correctly.`,
```
Comment (michaelhoefer) [inline messages/list.js:6]:
```suggestion
  Please make sure your machine has Java 8 or greater setup correctly.`,
```

File: messages/run.js [hunk 7]
```diff
@@ -1,16 +1,26 @@
...
+  "commandDescriptionLong": `Scan codebase with all rules by default 
+  or with a chosen set of rules if using rulename/category/ruleset filters.
+  You can choose the format of output and decide between printing the results directly
+  or as contents of a file that you provide with --outfile flag.
+  Please make sure your machine has Java8 or greater setup correctly.`,
   "flags": {
     "rulenameDescription": "[Description of 'rulename' parameter]",                   // TODO: Change this once the flag is implemented.
-    "categoryDescription": "One or more categories of rules to run. Multiple values can be specified as a comma-separated list.",
-    "rulesetDescription": "One or more rulesets to run. Multiple values can be specified as a comma-separated list.",
+    "categoryDescription": "categor(ies) of rules to run",
```
Comment (michaelhoefer) [inline messages/run.js:10]:
```suggestion
    "categoryDescription": "One or more categories of rules to run",
```

**Comment by rmohan20** [comment]
> Is this PR to include changes from Emily? That would be nice, to take her changes in one batch.

@michaelhoefer  This PR is just to create LongDescriptions so that Emily has something to start with and fix. Though this feedback loop seems tedious. Just spoke to her and I'm moving the messages to a document for her to review and modify. Once the text looks good, I'll be porting them back to the files.

File: messages/add.js (hunk 1 above)
Comment (rmohan20) [reply messages/add.js:10]: Let me fix this.

**Pull request merged**